src/Microsoft.DotNet.Build.VstsBuildsApi/VstsBuildHttpClient.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.VstsBuildsApi
+{
+    internal class VstsBuildHttpClient : IVstsDefinitionHttpClient
```

Comment:
No good reason. I think I may have originally had the clients inherit `JsonHttpClient` and didn't rethink the interface when I changed that. An abstract base class does sound better.